coap/api/transport.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"net"
+
+	mux "github.com/dereulenspiegel/coap-mux"
+	gocoap "github.com/dustin/go-coap"
```

Comment:
`coap`, not `gocoap`, was used before - please stay consistent (change one or another)

Reply:
One `coap` is ours, one is library package. For that reason, I can't use `coap` name. To stay consistent, I will use `gocoap` name in other packages too.